docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -326,6 +326,13 @@ You can resolve this on Debian-based systems, for example, by running::
     $ apt-get install locales
     $ dpkg-reconfigure locales
 
+You can resolve this for macOS systems by configuring your shell's locale::
```

Comment:
I would write "Mac OS". macOS seems to be a thing, but I've never seen this spelling (although I haven't used Mac since the 1990s).